executors/cmdexec/cmdexec.go

```diff
@@ -35,6 +40,31 @@ type CmdExecutor struct {
 	BackupLVM      bool
 }
 
+func (c *CmdExecutor) glusterCommand() string {
+	return fmt.Sprintf("gluster --mode=script --timeout=%v", c.GlusterCliTimeout())
+}
+
+func setWithEnvVariables(config *CmdConfig) {
+	var env string
+
+	env = os.Getenv("HEKETI_GLUSTER_CLI_TIMEOUT")
+	if env != "" {
+		value, err := strconv.ParseInt(env, 10, 32)
```

Comment:
can we use `ParseUint` instead of `ParseInt`?

Reply:
Good point. Will update the PR shortly.

Alternatively, we could discuss whether we want to just use `int` instead of `uint32` ...